### ISSUE TITLE
Populate default extension settings from windi.config.(js|ts) when it's available

### ISSUE THREAD

**Issue by joelmukuthu** [windicss/windicss-intellisense#270]
Hi, thanks for all your work on WindiCSS! All the various tools in the ecosystem come together really nicely. I'd like to suggest a small improvement for the extension. These settings could be disabled by default when `attributify` is also disabled (or rather not enabled) in the Windi configs:

- `windicss.enableAttrUtilityCompletion`
- `windicss.enableAttrVariantCompletion`

Thanks again!

**Comment by capozzid** [comment]
Correct, and the list should appear only on HTMLAttributes elements. Using typescript is very useful when creating a react component because it allows to see the required props. However, even when "attributify" list shouldn't be part of a component props, this extension adds them anyway. 

And by the way, thanks for the work on Windicss, it is truly useful and fun to work with ! 👍
